buildSrc/src/main/kotlin/com/wire/kalium/plugins/CommonDokkaConfig.kt

```diff
@@ -44,7 +46,12 @@ fun Project.commonDokkaConfig() {
 
     plugins.apply("org.jetbrains.dokka")
     val rootProject = rootProject
+    rootProject.mkdir("build/$DOKKA_CACHE_DIR") // creating cache dir
```

Comment:
I tried to create the dir using proper gradle directory variables and properties, but didn't work.
Also didn't want to spend more time but the reference will work as build is default naming convention.